packages/the-org/src/loadTest/test-runner.js

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env node
+
+import { execSync } from 'child_process';
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+// Get current directory
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+// Create logs directory if it doesn't exist
+const logsDir = path.join(__dirname, 'logs');
+if (!fs.existsSync(logsDir)) {
+  fs.mkdirSync(logsDir, { recursive: true });
+}
+
+// Generate filename with timestamp
+const timestamp = new Date().toISOString().replace(/:/g, '-');
+const mainLogFile = path.join(logsDir, `scalability-analysis-${timestamp}.log`);
+
+// Create a header for the main log file
+fs.writeFileSync(
+  mainLogFile,
+  `AGENT SCALABILITY ANALYSIS\n` +
```

Comment:
let's make this a single JSX string instead of multiple and make this a function in a utils file taking in the data as params that way if we have other scaling tests we can reference this